### PR TITLE
Many enhancements to FAQ

### PR DESCRIPTION
**Pull request by Gitoffthelawn**
Hope you are doing well Marcel.  BTW, I sent you an email...

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -94,22 +94,22 @@ You need 3 packages (applications) enabled (use search in NetGuard to find them
...
-to apply the rules with the conditions '*Allow when screen is on*' and '*Block when roaming*'.
+to apply the rules with the conditions *'Allow when screen is on'* and *'Block when roaming'*.
 
 See [here](http://forum.xda-developers.com/showpost.php?p=65723629&postcount=1788) for more details.
 
 <a name="FAQ10"></a>
 **(10) Will you provide a Tasker plug-in?**
 
-If disabling NetGuard is allowed to Tasker, any application can disabled NetGuard too.
+No, because if Tasker is allowed to disable NetGuard, any application can disabled NetGuard.
```
Comment (adisib) [inline FAQ.md]: You forgot to change "any application can disabled NetGuard" to "any application can disable NetGuard"

Comment (Gitoffthelawn) [inline FAQ.md]: Thanks... I'm on it.

**Comment by adisib** [comment]
Other recommendations (it wont let me add everything to a review so I'll just put it here I guess?):

Line 131: "This notification can unfortunately not be removed" to either "This notification can, unfortunately, not be removed", "This notification unfortunately can not be removed", or "This notification unfortunately cannot be removed" (unfortunately is awkwardly used).

Line 138: "To avoid this problem, at least temporary, close all applications and/or services which may be running in the background" to "To avoid this problem, at least temporarily, close all applications and/or services which may be running in the background" (temporary to temporarily).

Line 223: Same as 131

Line 328: "Conditions like '*When screen is on*' are on the other hand convenient because they work automatic" to "Conditions like '*When screen is on*' are on the other hand convenient because they work automatically" (automatic to automatically)

Line 396: "Besides not being trivial to add, schedule are in my opion not a good idea" to "Besides not being trivial to add, schedules are in my opion not a good idea" (schedule to schedules)

**Comment by Gitoffthelawn** [comment]
@adisib Implemented.  Thank you for your review.

**Comment by M66B** [comment]
@adisib can you please contact me through this contact form?
https://contact.faircode.eu/
(select "*NetGuard donation*" and fill in your GibHub username as challenge)

**Comment by M66B** [comment]
Big thanks for these corrections!

**Comment by Gitoffthelawn** [comment]
@M66B You're welcome!  Glad to help.

Were you asking *me* to contact you, or @adisib?

I sent you a direct email, but I can resend using your contact form, if you like.

**Comment by M66B** [comment]
I already reply earlier today, please let me know if you didn't receive my reply.

**Comment by Gitoffthelawn** [comment]
Hmmm... I didn't receive it.  Would you mind resending?

**Comment by M66B** [comment]
Done

**Comment by Gitoffthelawn** [comment]
Hmmm... not receiving it.  Strange.  Received email from others.  I just sent you another email to try to figure it out.

**Comment by Gitoffthelawn** [comment]
Got it.

**Pull request merged**